update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -12,55 +14,57 @@
     MySQLAdaptor dbaTestReactome = null;
     MySQLAdaptor dbaGkCentral = null;
     long authorIdTR = 0;
     long authorIdGK = 0;
-    boolean testMode = true;
 
     // Properties file should contain information needed to access current Test Reactome and GKCentral databases
+    String releaseNumber = "";
     try 
     {
       Properties props = new Properties();
       props.load(new FileInputStream(pathToConfig));
 
-      String userTR = props.getProperty("userTR");
-      String userGK = props.getProperty("userGK");
-      String passwordTR = props.getProperty("passwordTR");
-      String passwordGK = props.getProperty("passwordGK");
-      String hostTR = props.getProperty("hostTR");
-      String hostGK = props.getProperty("hostGK");
-      String databaseTR = props.getProperty("databaseTR");
-      String databaseGK = props.getProperty("databaseGK");
-      authorIdTR = Integer.valueOf(props.getProperty("authorIdTR"));
-      authorIdGK = Integer.valueOf(props.getProperty("authorIdGK"));
-      int port = Integer.valueOf(props.getProperty("port"));
-      if (props.getProperty("testMode") != null) {
-        testMode = Boolean.valueOf(props.getProperty("testMode"));
-      }
+      String userTR = props.getProperty("release.database.user");
+      String userGK = props.getProperty("curator.database.user");
+      String passwordTR = props.getProperty("release.database.password");
+      String passwordGK = props.getProperty("curator.database.password");
+      String hostTR = props.getProperty("release.database.host");
+      String hostGK = props.getProperty("curator.database.host");
+      String databaseTR = props.getProperty("release_current.name");
+      String databaseGK = props.getProperty("curator.database.name");
+      authorIdTR = Integer.valueOf(props.getProperty("personId"));
+      authorIdGK = Integer.valueOf(props.getProperty("personId"));
+      int portTR = Integer.valueOf(props.getProperty("release.database.port"));
+      int portGK = Integer.valueOf(props.getProperty("curator.database.port"));
+      releaseNumber = props.getProperty("releaseNumber");
+      //if (props.getProperty("testMode") != null) {
+      //  testMode = Boolean.valueOf(props.getProperty("testMode"));
+      //}
 
       // Set up db connections.
-      dbaTestReactome = new MySQLAdaptor(hostTR, databaseTR, userTR, passwordTR, port);
-      dbaGkCentral = new MySQLAdaptor(hostGK, databaseGK, userGK, passwordGK, port);
+      dbaTestReactome = new MySQLAdaptor(hostTR, databaseTR, userTR, passwordTR, portTR);
+      dbaGkCentral = new MySQLAdaptor(hostGK, databaseGK, userGK, passwordGK, portGK);
     } catch (Exception e) {
       e.printStackTrace();
     }
       UpdateDOIs.setAdaptors(dbaTestReactome, dbaGkCentral);
       logger.info("Starting UpdateDOIs");
-      UpdateDOIs.findAndUpdateDOIs(authorIdTR, authorIdGK, pathToReport, testMode);
+      UpdateDOIs.findAndUpdateDOIs(authorIdTR, authorIdGK, pathToReport, releaseNumber, testMode);
```

Comment:
If the catch block is triggered, the state of the variables being set from the property file is likely inconsistent.  Maybe the program should be terminated if that occurs instead of printing a stack trace and continuing?